apps/web/src/pages/templates/TemplatesListNoData.tsx

```diff
@@ -11,66 +16,158 @@ const NoDataHolder = styled.div`
+    width: 28px;
+    height: 28px;
+  }
 `;
 
 export const TemplatesListNoData = ({
-  onCreateClick,
-  onTryDigestClick,
-  tryDigestDisabled,
+  blueprints,
+  isLoading,
+  allTemplatesDisabled,
+  onBlankWorkflowClick,
+  onTemplateClick,
+  onAllTemplatesClick,
 }: {
-  onCreateClick: React.MouseEventHandler<HTMLButtonElement>;
-  onTryDigestClick: React.MouseEventHandler<HTMLButtonElement>;
-  tryDigestDisabled: boolean;
+  blueprints?: { id: string; name: string; description: string; iconName: IconName }[];
+  isLoading?: boolean;
+  allTemplatesDisabled?: boolean;
+  onBlankWorkflowClick: React.MouseEventHandler<HTMLButtonElement>;
+  onTemplateClick: (template: { id: string; name: string; description: string; iconName: IconName }) => void;
+  onAllTemplatesClick: React.MouseEventHandler<HTMLButtonElement>;
 }) => {
+  const [templateId, setTemplateId] = useState<string | null>(null);
+
   return (
     <NoDataHolder data-test-id="no-workflow-templates-placeholder">
-      <NoDataHeading>You don't have a workflow yet,</NoDataHeading>
-      <NoDataSubHeading>Start from scratch or play around with the Digest Demo Playground</NoDataSubHeading>
+      <NoDataSubHeading>Start from a blank workflow or use a template</NoDataSubHeading>
       <CardsContainer>
-        <Cards
-          cells={[
-            {
-              navIcon: PageGradient,
-              description: (
-                <Text size="lg" weight="bold">
-                  Create Workflow
-                </Text>
-              ),
-              onClick: onCreateClick,
-              testId: 'create-workflow-tile',
-            },
-            {
-              navIcon: DigestGradientIcon,
-              description: (
-                <Text size="lg" weight="bold">
-                  Try the Digest Playground
-                </Text>
-              ),
-              onClick: onTryDigestClick,
-              disabled: tryDigestDisabled,
-              testId: 'try-digest-playground-tile',
-            },
-          ]}
-        />
+        <Card data-test-id="create-workflow-tile" onClick={onBlankWorkflowClick}>
+          <FontAwesomeIcon icon={faFile} />
+          <span>Blank Workflow</span>
+        </Card>
+        {isLoading
+          ? Array.from({ length: 3 }).map((_, index) => (
+              <Card key={index} data-can-be-hidden={index === 2} data-test-id="second-workflow-tile">
+                <SkeletonIcon />
+                <Skeleton height={14} width="100%" />
+              </Card>
+            ))
+          : blueprints?.map((template, index) => (
+              <Popover
+                key={template.name}
+                opened={template.id === templateId}
+                withArrow
+                withinPortal
+                offset={5}
+                transitionDuration={300}
+                position="top"
+                width={300}
+                target={
+                  <Card
+                    data-can-be-hidden={index === 2}
```

Comment:
this `data-can-be-hidden` is a trick to hide the third tile on smaller screens ;) you can see that in styles